Cython/Utility/Builtins.c

```diff
@@ -657,3 +657,105 @@ static {{out_type}} __Pyx_PyMemoryView_Get_{{name}}(PyObject *obj) {
     return -1;
 }
 #endif
+
+//////////////////// PyInt_bit_count.proto ////////////////////
+
+static CYTHON_INLINE int __Pyx_PyInt_bit_count(PyObject *x); /*proto*/
+
+//////////////////// PyInt_bit_count ////////////////////
+
+static CYTHON_INLINE int __Pyx_PyInt_bit_count(PyObject *x) {
+#if defined(__has_builtin) && __has_builtin(__builtin_popcountll)
+    PY_LONG_LONG result;
+    int overflow;
+
+    result = PyLong_AsLongLongAndOverflow(x, &overflow);
+    if (overflow) {
+        return (int)PyLong_AsLong(CALL_UNBOUND_METHOD(PyLong_Type, "bit_count", x));
```

Comment:
This (and the call further down) needs error checking since the method call can fail.